aptos-move/framework/aptos-framework/sources/configs/stake.move

```diff
@@ -1009,7 +1043,8 @@ module aptos_framework::stake {
     ) acquires StakePool, StakePoolEvents, AptosCoinCapabilities, ValidatorConfig {
         let stake_pool = borrow_global_mut<StakePool>(pool_address);
         let validator_config = borrow_global<ValidatorConfig>(pool_address);
-        let num_missed_votes = *vector::borrow(&validator_perf.missed_votes, validator_config.validator_index);
+        let cur_validator_perf = *vector::borrow(&validator_perf.validators, validator_config.validator_index);
+        let num_missed_votes = cur_validator_perf.missed_votes;
```

Comment:
Don't you have to update the reward formula to only use proposals instead of missed votes?